### ISSUE TITLE
Strong reference caused bug

### ISSUE THREAD

**Issue by YaxinCheng** [siteline/SwiftUI-Introspect#26]
Hello, I really like your work, this is fantastic library to help building better views in SwiftUI. I am reporting a weird problem, and hopefully you can take a look

### Error message
```
InstrospectTest[86126:3366995] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: '<_TtGC7SwiftUI16PlatformViewHostGVS_P13$7fff3be90fe832PlatformViewRepresentableAdaptorGVS_P13$7fff3bede09022BridgedSystemTextFieldVS_20_TextFieldStyleLabel___ 0x10071ef20> has reached dealloc but still has a super view. Super views strongly reference their children, so this is being over-released, or has been over-released in the past.'
```

### My environment: 
- Xcode Version 11.4.1 (11E503a)
- Apple Swift version 5.2.2 (swiftlang-1103.0.32.6 clang-1103.0.32.51)
- SwiftUI-Introspect (0.1.0 installed from Swift Package Manager for master branch)

### My `body` part:

```swift
var body: some View {
    VStack {
      TextField("", text: $text)
        .font(.system(size: 32))
        .introspectTextField { textField in
          textField.font = .systemFont(ofSize: 32)
        }// no crash would ever happen if remove this .introspectTextField 
      ForEach(visible.startIndex ..< visible.endIndex, id: \.self) { index in
        VStack {
        Text("\(self.content[index])")
        Text("\(index)")
        }
      }.frame(width: 300, alignment: .top)
        .onReceive(keyPublisher, perform: keyPressed(_:))
    }
  }
```

### Why I think it's a problem of Introspect
When it crashes, the very last stack trace I can see from debugger is line 198 of `Introspect.swift`

Besides, if I remove this `.introspectTextField` function call, the problem will be gone. 

### Attempts made to narrow down problem
- Leave `.introspectTextField` block empty
  - Crash. Conclusion: the `.introspectTextField` function crashed the program, instead of what's in the call block
- Remove `.introspectTextField` entirely
  - No crash
- Remove `Text("\(self.content[index])")` from `VStack`
  - No crash
- Remove `Text("\(index)")` from `VStack`
  - No crash
- Combine `Text("\(self.content[index])")` and `Text("\(index)")` in `VStack` to one `Text`
  - No crash

### How to reproduce
I have packed my [test code](https://github.com/siteline/SwiftUI-Introspect/files/4499726/InstrospectTest.zip) here, download and unzip, and open with Xcode. Then run it, and type any number in the TextField. Use down arrow key to scroll down to the bottom, and up arrow to go up. It should crash near the bottom

**Comment by YaxinCheng** [comment]
This looks like a problem caused by wrongly using `VStack` instead of `List`. As I replaced the `VStack` to `List`, this problem is fixed. Closing the issue

**Comment by rlziii** [comment]
I also experienced this crash in a macOS application that I was developing which trying to use `.introspectTextField`.  Unfortunately, it seems like it was very specific to the SwiftUI view hierarchy that I was building.  If I tinkered around by adding or removing some (unrelated) views, the crash would either go away or come back.  The crash was on the same line (`Introspect.swift` on `198`).  I wasn't able to make a separate project that conveyed the crash, unfortunately, as perhaps it was very specific to some underlying view containers that SwiftUI was using.  If I find some extra time I will try to work on creating a project that consistently recreates the crash.  Unlike @YaxinCheng, I did not find a solution: I had to wrap an `NSTextField` in a `NSViewRepresentable` instead.
